### ISSUE TITLE
Circles doesn't follow design guidelines

### ISSUE THREAD

**Issue by depascalis** [FortAwesome/Font-Awesome#18340]
**Describe the bug**
_This is more of a question, but might be a bug._

The [Icon Design Guidelines](https://fontawesome.com/v5.15/how-to-use/on-the-web/using-kits/icon-design-guidelines) states that circles should be `15px` in diameter. When looking at FA icons, all the circles I have inspected are `15.5px` in diameter (eg. [circle regular](https://fontawesome.com/v5.15/icons/circle?style=regular), [circle solid](https://fontawesome.com/v5.15/icons/circle?style=solid), [info-circle](https://fontawesome.com/v5.15/icons/info-circle?style=solid)). There is probably a good reason for this so I'm curious of why that is?

I suggest clarifying the documentation on how to align the vertices open and closed paths on the 16px grid in the design phase, and also how strokes should be aligned on shapes (inner, center, outer).

**Version and implementation**
Version: 5.15

- [x] SVG with JS
- [x] Web Fonts with CSS
- [x] SVG Sprites
- [x] On the Desktop

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by sensibleworld** [comment]
Yup! Those docs need updating. I'm putting it on my to-do list. With version 6, our circles are all sized at 16px. With version 5, they were 15.5px... but something must have gotten lost in translation when we were first writing the docs.
